content/influxdb/cloud/notebooks/downsample.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Downsample data with notebooks
+description:
+weight: 102
+influxdb/v2.0/tags:
+menu:
+  influxdb_2_0:
+    name: Downsample data with notebooks
+    parent: Notebooks
+---
+{{% note %}}
+**Notebooks is currently an early access feature.**
+[Submit a request](https://w2.influxdata.com/notebooks-early-access/ ) to be added to the queue, and we will send you a confirmation when you’ve been added to early access.
+{{% /note %}}
+
+Downsample data to reduce the overall disk usage as data collects over time. This example walks through creating a notebook that downsamples data by aggregating data within windows of time, then storing the aggregate value in a new bucket.
+
+## Requirements
+
+- This example uses [InfluxDB Cloud demo data](/influxdb/cloud/reference/sample-data/#influxdb-cloud-demo-data).
+- Create a destination bucket to write downsampled data to. For details, see [Create a bucket](/influxdb/cloud/organizations/buckets/create-bucket/).
+
+## Downsample data with a notebook
+
+1. Create a new notebook (see [Create a notebook](/influxdb/cloud/notebooks/create-notebook/).
+2. Add a **Metric Selector** cell to select a bucket to query data from:
+  - In the **Choose a bucket** dropdown, select **Website Monitoring Bucket**.
+  - Select the **response_time** field.
+  - Select the **mean** function from the aggregate selector in the upper-right.
+3. Add a **Flux Transformation** cell with the following script to bring in data from the previous cell and downsample it:
+  ```sh
+  __PREVIOUS_RESULT__
+    |> aggregateWindow(fn: mean, every: 1h)
```

Comment:
We may want to add a note after this codeblock – If the `every` duration is longer the the total time range queried, aggregateWindow will only return one value.